### PR TITLE
Issue #11237: Support HTTPS URIs in acmeCA-2.0

### LINKED ISSUE #11237: acmeCA-2.0: Add SSL handler for connecting to CAs
Currently, we can connect to Pebble as that is built into acme4J. We need to add a handler for general use.

@jvanhill 

#9017

### PR DESCRIPTION
**Pull request by jvanhill**
OpenLiberty Pull Requester,

fixes #11237

### CODE REVIEW

**Comment by jvanhill** [comment]
#build

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -63,3 +63,18 @@ accountKeyFile.desc=A path to the file containing a key identifier for a registe
...
+acmeTransportConfig.desc=ACME transport layer configuration.
+
+sslProtocol=SSL protocol
+sslProtocol.desc=The SSL handshake protocol. Protocol values can be found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.
+
+trustStore=Truststore
+trustStore.desc=A keystore containing trusted certificate entries used by SSL for signing verification.
+
+trustStorePassword=Truststore password
+trustStorePassword.desc=The password used to load the truststore file. The value can be stored in clear text or encoded form. Use the securityUtility tool to encode the password.
```
Comment (Rwalls1) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Change "The password used to load the truststore file" to "The password that is used to load the truststore file".

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -63,3 +63,18 @@ accountKeyFile.desc=A path to the file containing a key identifier for a registe
...
 domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist, a new key is generated and written to this file. Back this file up to maintain control of the domain. 
+
+acmeTransportConfig=ACME transport configuration
+acmeTransportConfig.desc=ACME transport layer configuration.
+
+sslProtocol=SSL protocol
+sslProtocol.desc=The SSL handshake protocol. Protocol values can be found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.
+
+trustStore=Truststore
+trustStore.desc=A keystore containing trusted certificate entries used by SSL for signing verification.
```
Comment (Rwalls1) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Change "A keystore containing trusted certificate entries used by SSL" to A keystore that contains trusted certificate entries that are used by SSL"

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -63,3 +63,18 @@ accountKeyFile.desc=A path to the file containing a key identifier for a registe
 
 domainKeyFile=Domain key file
 domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist, a new key is generated and written to this file. Back this file up to maintain control of the domain. 
+
+acmeTransportConfig=ACME transport configuration
+acmeTransportConfig.desc=ACME transport layer configuration.
+
+sslProtocol=SSL protocol
+sslProtocol.desc=The SSL handshake protocol. Protocol values can be found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.
```
[the remark this answers is not in the document]
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made requested changes.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made requested changes.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made requested changes.

**Comment by jvanhill** [comment]
@Rwalls1 - I have made the requested messaging changes.

**Comment by jvanhill** [comment]
#run-libby

**Pull request merged**